### PR TITLE
FE-25 Add aws premium support request form

### PR DESCRIPTION
**Pull request by jonambas**
New route: `/support/aws-premium`

Available to aws users, and accessible through the billing summary page.

### CODE REVIEW

File: src/pages/billing/components/Banners.js [hunk 1]
```diff
@@ -48,20 +48,21 @@ export const ManuallyBilledBanner = ({ account, ...rest }) => {
...
     </Banner>
   );
 };
 
 /**
  * Premium Addon Plan CTA
  */
-export const PremiumBanner = () => (
+export const PremiumBanner = ({ isAWSAccount }) => (
   <Banner title='Premium Addon Plan' action={{
```
Comment (rajumsys) [inline src/pages/billing/components/Banners.js]: i think it's probably cleaner to create two buttons and pass that one conditionally. also should the track event be different as this is really different flow?

**Comment by jonambas** [comment]
@rajumsys i left traffic types as a field, not sure if there are more options other than `marketing, newletters, e-commerce, transactional`.

File: src/pages/support/components/tests/__snapshots__/PremiumSupportFields.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  />
+  <Field
+    component={[Function]}
+    label="Please provide the location of your website?"
+    name="website"
+    validate={[Function]}
+  />
+  <Field
+    component={[Function]}
+    label="If small service provider, please provide the location of your sending policy"
```
Comment (rajumsys) [inline src/pages/support/components/tests/__snapshots__/PremiumSupportFields.test.js.snap]: probably can say `If you're small service provider`

File: src/pages/support/PremiumSupportPage.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  componentDidUpdate (prevProps) {
+    const { submitSucceeded, showAlert, history, ticketId, createError } = this.props;
+
+    if (!prevProps.submitSucceeded && submitSucceeded && !createError) {
+      history.push('/landing');
+      return showAlert({
+        type: 'success',
+        message: `Request Submitted - Ticket # ${ticketId}`,
+        autoDimiss: false
```
Comment (rajumsys) [inline src/pages/support/PremiumSupportPage.js]: `autoDismiss`. This is why it was automatically dismissing!

File: src/pages/support/PremiumSupportPage.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    message: generateMessage(values),
+    issueType: 'Vetting',
+    subject: 'AWS Premium Vetting'
+  })
+
+  componentDidUpdate (prevProps) {
+    const { submitSucceeded, showAlert, history, ticketId, createError } = this.props;
+
+    if (!prevProps.submitSucceeded && submitSucceeded && !createError) {
+      history.push('/landing');
```
Comment (rajumsys) [inline src/pages/support/PremiumSupportPage.js]: as users will be coming to this from billing page, i guess they should go back to billing page.

**Pull request merged**